samples/grpc_server/route_guide.pb.go

```diff
@@ -1,36 +1,16 @@
-// Copyright 2015 gRPC authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
```

Comment:
Not sure if we can delete this and not sure how it was generated in the first place  :thinking:

Reply:
I restored it but the latest generator doesn't add it so I thought they decided to not add it anymore